tests/test_helper.tcl

```diff
@@ -783,6 +783,32 @@ proc close_replication_stream {s} {
     r config set repl-ping-replica-period 10
 }
 
+proc read_from_aof {fp} {
```

Comment:
i think these are more suitable to util.tcl rather than test_helper.tcl
(one of the benefits is that they can be used in the cluster tests one day)

Reply:
I put them here because their replication stream counterpart `read_from_replication_stream` and `assert_replication_stream` are in this file.

Moved.

Reply:
ohh, that's a good reason.
maybe one day we should move read_from_replication_stream too.
for now, i guess that we can keep the old one misplaced and the new one in the right place.